src/components/schedule/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./style.sass";
+import moment from "moment";
+
+function Schedule({ name, schedules }) {
+  return (
+    <article className="schedule-component-container">
+      <section className="cinema-name">{name}</section>
+      <section className="schedule-bound">
+        {schedules &&
```

Comment:
```suggestion
        {schedules.length &&
```
nếu map thì check length e nhé